### PR TITLE
Documentation/index.md: add Feedback section (Ralim#1552) as suggested

### PR DESCRIPTION
**Pull request by ia**
* **Please check if the PR fulfills these requirements**
- [x] The changes have been tested locally
- [x] There are no breaking changes

* **What kind of change does this PR introduce?**
Update `Documentation/index.md` which is exported as [ReadTheDocs online documentation](https://ralim.github.io/IronOS/).

* **What is the current behavior?**
As described in [this report](https://github.com/Ralim/IronOS/issues/1552):Ralim#1552, if a user opens the main online documentation page, there is no any information about _feedback_ to the project nor how to report a bug or ask community for help.

* **What is the new behavior (if this is a feature change)?**
if a user opens the main online documentation page, there is a clear "feedback welcome" section with direct links to _github Issue report form of the project_ and _direct link to Discussions board_.

* **Other information**:
Very good & smart suggestion BTW.

### CODE REVIEW

**Review by Ralim — APPROVED** [review]

**Comment by ia** [comment]
TL; DR: as far as I understand [nuances of github](https://github.com/orgs/community/discussions/26698#discussioncomment-3252952), you should change some protection settings for branch `dev` in the original `IronOS` repo. Otherwise these stalled decommissioned jobs still will be in a permanent waiting status (which in its turn makes automerge is not possible).

It seems that enabling `auto-merge` won't do `merge` automatically. Although it says:

> This pull request will squash and merge automatically when all requirements are met.

There is a still that little issue due to renaming `jobs` steps. The thing is that I did this pull-request against forked repo with fixed `push.yml` (so the file is a full equivalent between original & forked repo). As far as I understand the comment from the link above:

> When you rename or remove jobs, you need to update any branch protection rules that use them.

In this case you should do something [here](https://github.com/Ralim/IronOS/settings/branches) for `dev` (probably manually disabling/removing/re-adding jobs manually or something like that).

I'm terribly sorry for this inconvenience. If I would know that github works in such strange illogical way with renamed jobs, I would keep them as is. Sorry. But on top of that, if we go to `Actions` tab, [**it even doesn't show**](https://github.com/Ralim/IronOS/actions/runs/5580246805?pr=1744) those ghost jobs from previous `push.yml` revision (as it should). :man_shrugging:

**Comment by Ralim** [comment]
Hahahah yeah dont stress I'm on it when I get to a pc later on.

**Comment by ia** [comment]
Just a suggestion but I guess now you can close as fixed [this](https://github.com/Ralim/IronOS/issues/1552):Ralim#1552 and [this](https://github.com/Ralim/IronOS/issues/1643):Ralim#1643 issues for actualization of issue tracker BTW.
